### PR TITLE
Readme update from issue #1

### PR DESCRIPTION
**Pull request by MalonzaElkanah**
Worked on Installation guide and Running guide.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,124 @@
...
+##### Create a new issue
+
+If you spot a problem with the project, search if a similar issue already exists. 
+If a related issue doesn't exist, you can open a new issue 
+
+##### Solve an issue
+
+Scan through our existing issues to find one that interests you. 
+You can narrow down the search using `labels` as filters. 
+If you find an issue that you want to work on, comment on it to have it assigned to you;
```
Comment (Barissa-Imran) [inline README.md:52]: - Suggest collaborators create their own issue and upon approval work on it
- Suggest that collaborators should only claim an issue when they are familiar with the code base and can work on it

File: README.md [hunk 2]
```diff
@@ -1,2 +1,124 @@
...
+##### Make changes locally
+1. Fork the repository.
+- Using GitHub Desktop:
+  - [Getting started with GitHub Desktop](https://docs.github.com/en/desktop/installing-and-configuring-github-desktop/getting-started-with-github-desktop) will guide you through setting up Desktop
+  - Once Desktop is set up, you can use it to [fork the repo](https://docs.github.com/en/desktop/contributing-and-collaborating-using-github-desktop/cloning-and-forking-repositories-from-github-desktop)!
+
+- Using the command line:
+  - [Fork the repo](https://docs.github.com/en/github/getting-started-with-github/fork-a-repo#fork-an-example-repository) so that you can make your changes without affecting the original project until you're ready to merge them
+
+2. Create a branch and start working on your changes!
```
Comment (Barissa-Imran) [inline README.md:64]: - collaborators should rebase the pre-dev branch when creating branches. (This is where new code will be merged)
```bash
git branch newbranch pre-dev
```
or
```bash
git checkout -b newbranch pre-dev
```
or 

on GitHub, select the pre-dev branch in your fork then create a branch (it should tell you that you're rebasing the pre-dev branch)

**Review by Barissa-Imran — CHANGES_REQUESTED** [review]

**Comment by Barissa-Imran** [comment]
Hi @MalonzaElkanah , I have made some comments on things to add.

Otherwise LGTM

File: README.md (hunk 1 above)
Comment (jumalaw98) [reply README.md:52]: Sorry for late review of this, am glad for your update

**Comment by jumalaw98** [comment]
@Barissa-Imran @MalonzaElkanah  can you make update, on the collaboration part: 

Forking the main repo
Check out `pre-dev` branch and pull the updates from it which is the collaboration branch 

Whats your take on this

**Comment by Barissa-Imran** [comment]
This should work but increases errors and the number of commits from collaborators and repo managers. Have to sync the repos which can be problematic when main is ahead of the development branches. Creates a loop of commits. (after committing you're asked to update the branch and loops like that). But I don't know

**Pull request merged**